content/docs/rfcs/13/README.md

```diff
@@ -122,7 +122,7 @@ RPC call to query historical messages.
   Leaving this field empty means no filter on the content topic of message history is required.
   This field SHOULD be left empty in order to retrieve historical waku messages regardless of their content topics.
 - `PagingInfo` holds the information required for pagination.  Its `pageSize` field indicates the number of  `WakuMessage`s to be included in the corresponding `HistoryResponse`. If the `pageSize` is zero then no pagination is required. If the `pageSize` exceeds a threshold then the threshold value shall be used instead. In the forward pagination request, the `messages` field of the `HistoryResponse` shall contain at maximum the `pageSize` amount of waku messages whose `Index` values are larger than the given `cursor` (and vise versa for the backward pagination). Note that the `cursor` of a `HistoryQuery` may be empty (e.g., for the initial query), as such, and depending on whether the  `direction` is `BACKWARD` or `FORWARD`  the last or the first `pageSize` waku messages shall be returned, respectively.
-The queried node MUST sort the `WakuMessage`s based on their `Index`, where the `receivedTime` constitutes the most significant part and the `digest` comes next, and then perform pagination on the sorted result. As such, the retrieved page contains an ordered list of `WakuMessage`s from the oldest message to the most recent one.
+The queried node MUST sort the `WakuMessage`s based on their `Index`, where the `timestamp` constitutes the most significant part and the `digest` comes next, and then perform pagination on the sorted result. As such, the retrieved page contains an ordered list of `WakuMessage`s from the oldest message to the most recent one.
```

Comment:
Do we definitely want this to be a MUST and change this? Vs accommodating both but recommending one over the other

Reply:
Using sender timestamp by default also seems like it'd have security implications that we should highlight, e.g. arbitrary data

Reply:
Thanks for the comments @oskarth! 
I can integrate both fields i.e. receiver and sender timestamp for the pagination, but it would add more complexity as to which field should and should not be considered/ignored during the paging.  I was not sure whether such complication would be desirable and necessary and would have future use-case or not. 

Moreover, when the use of sender/receiver time becomes optional, the cursor of one store node might not be useful for another store node (we had some interoperability and consistency scenarios across different store nodes back then). e.g., if querying nodes want to use their cursor across different store nodes then the receiver timestamp of the cursor with high probability would be inconsistent. 

One more thing, if we are considering adding other query criteria to the pagination part e.g.,`blockheight`, it means we are going to change the store specs in the future, right? then wouldn't that mean the store protocol specs is not going to be stable any soon?
One of my aims was to avoid such optional fields (for which I have no clear usecase in mind) to get the store specs to a stable mode.

Reply:
@oskarth 
I have updated the specs, now the cursor embodies both the sender and the receiver times. However, only the sender time is used for sorting of the waku messages.